.bazelrc

```diff
@@ -44,13 +44,20 @@ build:linux_x86_64 --host_copt=-march=haswell
 build:linux_aarch64 --copt=-march=armv8-a
 build:linux_aarch64 --host_copt=-march=armv8-a
 
+# Configure for Linux on ARM64 (Apple Silicon)
+# Macosx version in plat name should be aligned with --macos_minimum_os
```

Comment:
plan_name where? Clarify the comment so that it is clear. I mean add the file name, and the property is named `plat_name`